installer.sh.in

```diff
@@ -814,8 +819,14 @@ set_bootloader() {
 }
 
 test_network() {
-    rm -f xtraeme.asc && \
-        xbps-uhelper fetch http://alpha.de.repo.voidlinux.org/live/xtraeme.asc >$LOG 2>&1
+    # Pick a peace of data to download to ensure that a network connection is
+    # available.
+    local _file="http://alpha.de.repo.voidlinux.org/live/xtraeme.asc"
+
+    # Download with XBPS, remove it directly afterwards
+    xbps-uhelper fetch "$file" >$LOG 2>&1 || rm -f xtraeme.asc
```

Comment:
doesn't this change the behavior?

Reply:
Yes, your perfectly right! That was my fault, sorry. I'll fixed it.
I'll introduced a new variable `local` in the test_network function to capture the status of the download, but remove the (probably) downloaded file immediately afterwards to get rid of this file if you abort the installation. I'll think this is a much cleaner solution than leave the `xtraeme.asc` file in the home directory which might confusing some users.